frontend/src/components/RequestDialog.vue

```diff
@@ -132,7 +137,10 @@ const submitRequestTransaction = async (formResult: {
     targetChainId: Number(formResult.toChainId.value),
     sourceTokenAddress: formResult.tokenAddress.value,
     sourceChainId: Number(formResult.fromChainId.value),
-    targetTokenAddress: formResult.tokenAddress.value,
+    targetTokenAddress: getTargetTokenAddress(
+      formResult.toChainId.value,
+      formResult.tokenAddress.label,
```

Comment:
the function takes the symbol, not the address as parameter

Reply:
this's a select box and the token symbol is the `label`, the address would be `formResult.tokenAddress.value` instead!